Running FedStream with: Dir=0.8, Init_Class=10, Epoch=20
has read
  0%|          | 0/100 [00:00<?, ?it/s]  1%|          | 1/100 [05:31<9:07:18, 331.71s/it]  2%|▏         | 2/100 [10:56<8:55:17, 327.73s/it]  3%|▎         | 3/100 [16:23<8:49:24, 327.46s/it]  4%|▍         | 4/100 [21:53<8:45:18, 328.32s/it]  4%|▍         | 4/100 [26:26<10:34:45, 396.72s/it]
Traceback (most recent call last):
  File "/home/kemove/lt/FedAlgorithm/alg/FedStream.py", line 775, in <module>
    alg.online_train()
  File "/home/kemove/lt/FedAlgorithm/alg/FedStream.py", line 738, in online_train
    global_loss_list, accuracy_list = self.alg.run(
                                      ^^^^^^^^^^^^^
  File "/home/kemove/lt/FedAlgorithm/alg/FedDyn.py", line 134, in run
    ].local_update_dyn(
      ^^^^^^^^^^^^^^^^^
  File "/home/kemove/lt/FedAlgorithm/alg/Clients.py", line 286, in local_update_dyn
    for batch in dataloader:
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 675, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torch/utils/data/dataset.py", line 348, in __getitem__
    return self.datasets[dataset_idx][sample_idx]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^
  File "/home/kemove/lt/FedAlgorithm/alg/Clients.py", line 37, in __getitem__
    data = self.trans(self.local_data[index])
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torchvision/transforms/transforms.py", line 95, in __call__
    img = t(img)
          ^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torchvision/transforms/transforms.py", line 137, in __call__
    return F.to_tensor(pic)
           ^^^^^^^^^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/torchvision/transforms/functional.py", line 168, in to_tensor
    img = torch.from_numpy(np.array(pic, mode_to_nptype.get(pic.mode, np.uint8), copy=True))
                                                            ^^^^^^^^
  File "/home/kemove/anaconda3/envs/test/lib/python3.11/site-packages/PIL/Image.py", line 523, in mode
    @property

KeyboardInterrupt
